certbot/tests/account_test.py

```diff
@@ -18,46 +20,60 @@
 import certbot.tests.util as test_util
 
 
-KEY = jose.JWKRSA.load(test_util.load_vector("rsa512_key.pem"))
-
-
 class AccountTest(unittest.TestCase):
     """Tests for certbot._internal.account.Account."""
 
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.KEY = jose.JWKRSA.load(test_util.load_vector("rsa512_key.pem"))
+
     def setUp(self):
         from certbot._internal.account import Account
         self.regr = mock.MagicMock()
         self.meta = Account.Meta(
             creation_host="test.certbot.org",
             creation_dt=datetime.datetime(
                 2015, 7, 4, 14, 4, 10, tzinfo=pytz.UTC))
-        self.acc = Account(self.regr, KEY, self.meta)
+        self.acc = Account(self.regr, self.KEY, self.meta)
         self.regr.__repr__ = mock.MagicMock(return_value="i_am_a_regr")
 
         with mock.patch("certbot._internal.account.socket") as mock_socket:
             mock_socket.getfqdn.return_value = "test.certbot.org"
             with mock.patch("certbot._internal.account.datetime") as mock_dt:
                 mock_dt.datetime.now.return_value = self.meta.creation_dt
-                self.acc_no_meta = Account(self.regr, KEY)
+                self.acc_no_meta = Account(self.regr, self.KEY)
 
     def test_init(self):
         self.assertEqual(self.regr, self.acc.regr)
-        self.assertEqual(KEY, self.acc.key)
+        self.assertEqual(self.KEY, self.acc.key)
         self.assertEqual(self.meta, self.acc_no_meta.meta)
 
     def test_id(self):
-        self.assertEqual(
-            self.acc.id, "7adac10320f585ddf118429c0c4af2cd")
+        self.assertEqual(self.acc.id, "7adac10320f585ddf118429c0c4af2cd")
 
     def test_slug(self):
-        self.assertEqual(
-            self.acc.slug, "test.certbot.org@2015-07-04T14:04:10Z (7ada)")
+        self.assertEqual(self.acc.slug, "test.certbot.org@2015-07-04T14:04:10Z (7ada)")
 
     def test_repr(self):
         self.assertTrue(repr(self.acc).startswith(
           "<Account(i_am_a_regr, 7adac10320f585ddf118429c0c4af2cd, Meta("))
 
 
+# class AccountTestECKey(AccountTest, test_util.ConfigTestCase):
```

Comment:
were these commented-out tests left in accidentally?

Reply:
So this is probably the place I'd need some help :)